website/static/js/comment.js

```diff
@@ -166,16 +182,33 @@ BaseComment.prototype.submitReply = function() {
         return;
     }
     self.submittingReply(true);
-    osfHelpers.postJSON(
-        nodeApiUrl + 'comment/',
+    var url = osfHelpers.apiV2Url('nodes/' + window.contextVars.node.id + '/comments/', {});
+    var request = osfHelpers.ajaxJSON(
+        'POST',
+        url,
         {
-            target: self.id(),
-            content: self.replyContent(),
-        }
-    ).done(function(response) {
+            'isCors': true,
+            'data': {
+                'data': {
+                    'type': 'comments',
+                    'attributes': {
+                        'content': self.replyContent()
+                    },
+                    'relationships': {
+                        'target': {
+                            'data': {
+                                'type': self.id() === undefined ? 'nodes' : 'comments',
```

Comment:
These lines get changed when I add in file comments: https://github.com/samanehsan/osf.io/blob/feature/file-comments-v2-merge/website/static/js/comment.js#L231-L232